src/schema-validator.ts

```diff
@@ -17,7 +17,13 @@ export const validatePayload = async ({
     const propertySchema = schemaValidator.schema[propertyToValidate]
 
     try {
-      await propertySchema?.yupSchema.validate(payload[propertyToValidate], propertySchema.validateOptions)
+      await propertySchema?.yupSchema.validate(payload[propertyToValidate], {
+        ...propertySchema.validateOptions,
+        context: {
+          req: payload,
```

Comment:
use payload instead of req as it can be anything, not necessarily an express request object